Refactor update beacon set mechanism

hiletmis opened the pull request:
Closes #182

bbenligiray reviewed (changes requested):
I'd like to keep the `signMessage()` flow as documentation. Check if the account is `0x07b589f06bD0A5324c4E2376d66d2F4F25921DE1`, use `signMessage()` if it is, use the hardcoded calldata otherwise.

hiletmis commented:
@bbenligiray including gas estimation signatures might not be a good idea. Removed it from script and updated the test call data for `nodary-rpc-watcher`
